src/app/_components/SocialLink/SocialLink.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import p from 'prop-types';
+import Link from 'next/link';
+import cn from '@utils/cn';
+
+export default function SocialLink({ items, className, status }) {
+  const { links } = items;
+  return status === 'footerSocials' ? (
+    <div className="flex gap-4">
+      {links
+        ?.sort((a, b) => {
+          if (a.title < b.title) {
+            return -1;
+          }
+          if (a.title > b.title) {
+            return 1;
+          }
+          return 0;
+        })
+        .map((link, idx) => (
+          <Link
```

Comment:
@Yivgha Let's refactor this component in a few steps.

1. First, let's make it dumb. Simply render links without any sorting or branching logic. Move the sorting logic into the `Footer` component.
2. Receive only the necessary props. In this case, accept only the links objects.
3. Replace the usage of `.sort` with `.toSorted` (or clone array before sorting, like [...arr].sort) since it doesn't create side effects by mutating the original array.

Reply:
Done.